_posts/2021-06-23-#22261.md

```diff
@@ -6,8 +6,8 @@ pr: 22261
 authors: [jnewbery]
 components: ["mempool", "p2p"]
 host: glozow
-status: upcoming
-commit:
+status: past
+commit: ba99f37
```

Comment:
The tagged branch needs to be pushed in order for the generated link (here, to https://github.com/bitcoin-core-review-club/bitcoin/tree/pr22261) to work. See CONTRIBUTING.md.

Reply:
Done!